fencing/commands.c

```diff
@@ -340,7 +363,7 @@ stonith_device_execute(stonith_device_t * device)
         crm_debug("Operation %s%s%s on %s now running with pid=%d, timeout=%ds",
                   cmd->action, cmd->victim ? " for node " : "", cmd->victim ? cmd->victim : "",
                   device->id, exec_rc, cmd->timeout);
-        device->active_pid = exec_rc;
+        device->active_pids = g_list_append(device->active_pids, GINT_TO_POINTER(exec_rc));
```

Comment:
FYI my personal preference would be to functionize adding/removing active pids, for readability and code re-use, but it's not important. Something like add_active_pid(device, exec_rc)